src/cargo/rust-ecosystem.md

```diff
@@ -44,3 +44,7 @@ Key points:
 [editions]: https://doc.rust-lang.org/edition-guide/
 
 </details>
+
+[^rustc]: Please note, it is quite rare to ever use the compiler directly not through `cargo`.
+
+[^cargo]: Cargo itself is a powerful and comprehensive tool capable of many advanced features including but not limited to: Project structure, [workspaces](https://doc.rust*lang.org/cargo/reference/workspaces.html), dev dependencies, [build scripting](https://doc.rust-lang.org/cargo/reference/build-scripts.html), [binary installation](https://doc.rust-lang.org/cargo/commands/cargo-install.html), etc. It is extensible with sub command plugins as well (such as [cargo clippy](https://github.com/rust-lang/rust-clippy) or [cargo binstall](https://github.com/cargo-bins/cargo-binstall)). Read more from the [official Cargo Book](https://doc.rust-lang.org/cargo/)
```

Comment:
I'm trying to move towards a style where I use reference links in Markdown. So something like the below. The advantage: less inline Markdown code for translators to deal with:

```suggestion
[^cargo]: Cargo itself is a powerful and comprehensive tool capable of many advanced features including but not limited to: Project structure, [workspaces], dev dependencies, [build scripting], [binary installation], etc. It is extensible with sub command plugins as well (such as [`cargo clippy`]). Read more from the [official Cargo Book].

[workspaces]: https://doc.rust*lang.org/cargo/reference/workspaces.html
[build scripting]: https://doc.rust-lang.org/cargo/reference/build-scripts.html
[binary installation]: https://doc.rust-lang.org/cargo/commands/cargo-install.html
[`cargo clippy`]: https://github.com/rust-lang/rust-clippy
[official Cargo Book]: https://doc.rust-lang.org/cargo/
```

(I dropped `cargo-binstall` since I haven't heard of it and I don't think we should include more than just the basics here)